runtimes/nodejs/object-write.js

```diff
@@ -48,9 +50,8 @@ async function main(args) {
 
 
 function getParamsCOS(args, COS) {
-  const bucket = args.bucket || args.Bucket;
-  const key = args.key || args.Key;
-  let body = args.body || args.Body;
+  const { bucket, key } = args;
+  let { body } = { args };
   if (body.type === 'Buffer') {
```

Comment:
same issue here, 
should be `let { body } = args;`

Reply:
thanks, should be last one.